kratos/includes/constitutive_law.h

```diff
@@ -82,6 +82,11 @@ class KRATOS_API(KRATOS_CORE) ConstitutiveLaw : public Flags
     typedef std::size_t SizeType;
     typedef Geometry<Node < 3 > > GeometryType;
 
+    typedef array_1d<double, 6> StrainVectorType;
+    typedef array_1d<double, 6> StressVectorType;
```

Comment:
```suggestion
    typedef array_1d<double, 6> StressVector3DType;
```

Reply:
i would not use the 3D, since it can be used with size 3,4 or 6